Music Advisor/task/src/advisor/controller/commands/FeaturedCommand.java

```diff
@@ -0,0 +1,26 @@
+package advisor.controller.commands;
+
+import advisor.model.DataSource;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+
+import java.util.*;
+
+import static advisor.controller.utils.ForTests.*;
+import static advisor.controller.utils.HttpHandler.doGETRequest;
+import static advisor.controller.utils.HttpHandler.isResponseValid;
+
+public class FeaturedCommand extends PlaylistsExtractor implements Command {
+
+    @Override
+    public boolean execute(final String address, DataSource source) {
+        String response = doGETRequest(address + "featured-playlists", source);
```

Comment:
Also as for me it will be better to use there constant 
private static final String FEATURED_PLAYLISTS_ADDRESS = "%s/featured-playlists" with String.format()
and do the same for every command